### DISCUSSION TITLE
connected user who subscribe realtime id

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#3577] [Q&A]
```
onRealtimeAfterSubscribeRequest((e) => {
    const record = e.client.get("authRecord")

    if (record && !record.get("online")) {
        record.set("online", true);
        $app.dao().saveRecord(record);
    }
})
```

here can we get the userinfo or id ?to identify which user connected?

**Comment by ganigeorgiev** [comment]
I'm not sure I follow.

If the subscription was submitted from an authenticated user then `e.client.get("authRecord")` should return it. To get the id you can call `record.id`.
